mailqueue/models.py

```diff
@@ -99,7 +100,8 @@ def _send(self):
 
             subject, from_email = self.subject, self.from_address
             text_content = self.content
-            msg = EmailMultiAlternatives(subject, text_content, from_email)
+            reply_to = self.reply_to
+            msg = EmailMultiAlternatives(subject, text_content, from_email, headers={"reply-to": reply_to})
```

Comment:
Sorry I didn't catch this before, but you'll want to add a conditional here to that blank ```reply-to``` headers aren't sent.

Something like...
```python
msg = EmailMultiAlternatives(subject, text_content, from_email)
if self.reply_to:
    msg.headers = {'reply-to': self.reply_to}
```